polymesh-docs/primitives/identity.mdx

```diff
@@ -139,3 +139,25 @@ The _portfolios_ granularity allows a key to be limited to interacting with only
 The access of a particular key is the intersection of each of these categories. It is possible to permission a key with full access to any of these categories.
 
 Not all on-chain actions are associated with either an asset, or a portfolio (for example staking). For such actions, these permission types are not checked.
+
+## Child Identities
+
+An identity as described above is always created by a [CDD service provider](../cdd/).
+
+This process involves two steps for CDD providers:
+  - creating an on-chain identity for the user, referenced via the returned DID
+  - issuing a CDD claim to the new on-chain identity
+
+When a user submits a transaction to the network, we:
+  - determine the associated identity by looking at what identity the key that signs the transaction is linked to (this can be at most one identity)
+  - check that this associated identity has a valid (issued by a current CDD provider, not expired) CDD claim
+
+However for some use-cases it may be helpful for existing identities created via CDD providers to be able to create additional child identities unilaterally (i.e. without the involvement of a CDD provider).
+
```

Comment:
Some use-cases:

1. Isolate a Smart contract from the deployer's identity to protect user funds held/controlled by the Smart Contract.
2. Extra layer of protection, with the child identities being "hot wallets" and the parent identity being "cold wallets".  Since assets are held at the identity-level, it can be useful to have hot/cold identities.
3. Organizational structure (one child identity for each department/sub-company).